packages/modal/src/use-modal.ts

```diff
@@ -1,5 +1,6 @@
 import { useIds } from "@chakra-ui/hooks"
-import { callAllHandlers, mergeRefs, PropGetter } from "@chakra-ui/utils"
+import { callAllHandlers, PropGetter } from "@chakra-ui/utils"
```

Comment:
Same question about exported types here as well 🤔

Reply:
Yeap! Anything that's specific to React shouldn't belong in `utils` anymore, types and all

Reply:
Thanks. Moved all react-based types to `react-utils` 👍